### ISSUE TITLE
Fabric.exe Continuously Using About 30% CPU on One Node Out of Five on Primary (Seed) Nodes After 7.1.417.9590 Update

### ISSUE THREAD

**Issue by brianstringfellow** [microsoft/service-fabric#983]
After updating our Service Fabric runtime to 7.1.417.9590 from 7.0.466.9590, the Fabric.exe is continuously using about 30% CPU on one node out of five on the primary (seed) nodes. The CPU consumption is observed on seven different clusters after each had been updated. Sometimes the node affected changes over time, such as when the underlying VM is restarted.

Example from task manager:
![image](https://user-images.githubusercontent.com/22823524/87335937-f3878d00-c4f5-11ea-975c-e5490d16b146.png)

CPU Usage Chart When Update Deployed:
![image](https://user-images.githubusercontent.com/22823524/87336371-93451b00-c4f6-11ea-9819-2b90fcf1a38a.png)

Is the higher CPU usage a known problem?

**Comment by aricamf** [comment]
I have the exact observation in my side. Fabric.exe started to use around 20-40% CPU since I upgraded to 7.1.417.9590.

**Comment by gkhanna79** [comment]
@brianstringfellow @aricamf Can you confirm if your cluster has applications that specify certificates for Endpoints? We had an issue in that space that has been fixed in the upcoming CU.

@dragav FYI

**Comment by brianstringfellow** [comment]
@gkhanna79 @dragav Yes, we have applications that specify certificates for endpoint. All of our frontend applications use port 443.

**Comment by gkhanna79** [comment]
Do you have performance counters for the node when the CPU is high?

**Comment by brianstringfellow** [comment]
@gkhanna79 
The only performance counters enabled are in the WadCfg:
```
"PerformanceCounters": {
  "PerformanceCounterConfiguration": [
    {
      "counterSpecifier": "\\Processor(_Total)\\% Processor Time",
      "sampleRate": "PT15S",
      "unit": "Percent"
    },
    {
      "counterSpecifier": "\\Memory\\Available MBytes",
      "sampleRate": "PT15S"
    }
  ]
}

```

It is interesting to see the higher CPU usage shift to other nodes over time.
![image](https://user-images.githubusercontent.com/22823524/89075520-41dcbe80-d333-11ea-955f-ccd56cccbc41.png)

**Comment by jagilber** [comment]
@brianstringfellow it may be related to known issue https://github.com/Azure/Service-Fabric-Troubleshooting-Guides/blob/master/Known_Issues/Service%20Fabric%207.1%20High%20CPU%20Fabric.exe%20One%20Node.md
